<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adc Gui</string>
  </property>
  <widget class="QGroupBox" name="groupBoxAdcChannelConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>ADC Channel Configuration</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>385</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelEventType">
         <property name="text">
          <string>Event Type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxEventType">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Below</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Above</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Outside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelEventPeriod">
         <property name="text">
          <string>Event Period:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditEventPeriod">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelThresLow">
         <property name="text">
          <string>Threshold Low:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditThresholdLow">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelThresHigh">
         <property name="text">
          <string>Threshold High:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditThresholdHigh">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonSetChannelConfig">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonGetChannelConfig">
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="labelAdcPort">
      <property name="text">
       <string>ADC Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxAdcPort"/>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBoxEnableAdc">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButtonOpenDevice">
      <property name="text">
       <string>Open Device</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelAdcChannel">
      <property name="text">
       <string>ADC Channel:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxAdcChannel"/>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBoxChannelEnableAdc">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelAdcValue">
      <property name="text">
       <string>ADC Value:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="AdcValue">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButtonGetAdcChannelValue">
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelAdcValues">
      <property name="text">
       <string>ADC Values:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="AdcValues">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButtonGetAdcValues">
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelResolution">
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBoxAdcResolution">
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButtonSetResolution">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButtonGetResolution">
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
